src/shared/views/ChangeDetection/InstructionsScreen.js

```diff
@@ -143,84 +145,99 @@ export default class CDInstructionsScreen extends React.Component<Props> {
-                            and the tile will turn red
+                            <Trans i18nKey="CDInstructionsScreen:badImagery">
+                                Imagery issue, like if either image has clouds
+                                covering the view?{' '}
+                                <Text style={{ fontWeight: 'bold' }}>
+                                    Tap three times
+                                </Text>{' '}
+                                and the tile will turn red
+                            </Trans>
                         </Text>
                     </View>
 
-                    <Text style={styles.tutParagraph}>
-                        If you need to see an image more closely, tap and hold
-                        the image and it&apos;ll zoom in a little more.
-                    </Text>
+                    <Text style={styles.tutParagraph}>{t('holdZoom')}</Text>
 
-                    <Text style={styles.header}>Hint:</Text>
+                    <Text style={styles.header}>{t('hint')}</Text>
                     <Text style={styles.tutParagraph}>
-                        Sometimes different imagery sources will have been used.
-                        The images may be aligned slightly differently or might
-                        be a different resolution. Remember, you&apos;re looking
-                        for{' '}
-                        <Text style={{ fontWeight: 'bold' }}>
-                            definite changes in settlements and buildings
-                        </Text>{' '}
-                        so if it looks like the same buildings are there, but
-                        maybe there&apos;s a new roof, then this would be a
-                        &quot;no change&quot; scenario and you&apos;d simply
-                        swipe to the next image.
+                        <Trans i18nKey="CDInstructionsScreen:differentImagery">
+                            Sometimes different imagery sources will have been
+                            used. The images may be aligned slightly differently
+                            or might be a different resolution. Remember,
+                            you&apos;re looking for{' '}
+                            <Text style={{ fontWeight: 'bold' }}>
+                                definite changes in settlements and buildings
+                            </Text>{' '}
+                            so if it looks like the same buildings are there,
+                            but maybe there&apos;s a new roof, then this would
+                            be a &quot;no change&quot; scenario and you&apos;d
+                            simply swipe to the next image.
+                        </Trans>
                     </Text>
                     <Text style={styles.header}>&nbsp;</Text>
                 </ScrollView>
             </View>
         );
     }
 }
+
+export default withTranslation('CDInstructionsScreen')(CDInstructionsScreen);
```

Comment:
I'm a bit confused by this file and don't know how it is used. It seems to be related to instructions for change detection, but also somehow to the tutorial. 
I thought that we still load information on the tutorial from firebase. Is this the first step towards moving the tutorial into the app and make it translatable?

Reply:
This is the static text that is in the help screen you get to when you tap the `i` icon at the top right corner of the screen. The content is then decided depending on whether you're in tutorial mode or not. And it should be translated from within transifex.
The rest of the tutorial content (the prompts that you see in the gray box in the middle of the imagery) has not changed yet, so it's still in firebase.